extensions/positron-python/src/client/interpreter/configuration/interpreterSelector/commands/setInterpreter.ts

```diff
@@ -737,6 +739,10 @@ function getGroup(item: IInterpreterQuickPickItem, workspacePath?: string) {
  * @returns A new filter function that includes the original filter function and the additional filtering logic
  */
 function filterWrapper(filter: ((i: PythonEnvironment) => boolean) | undefined) {
-    return (i: PythonEnvironment) => (filter ? filter(i) : true) && shouldIncludeInterpreter(i.path);
+    return (i: PythonEnvironment) =>
+        (filter ? filter(i) : true) &&
+        shouldIncludeInterpreter(i.path) &&
+        (i.version?.major ?? 0) >= MINIMUM_PYTHON_VERSION.major &&
+        (i.version?.minor ?? 0) >= MINIMUM_PYTHON_VERSION.minor;

```

Comment:
I came across some version checking code while working on https://github.com/posit-dev/positron/pull/6398 that we could reuse!

Thoughts on pulling the `isVersionSupported` check out into a util somewhere and reusing it here in setInterpreter? https://github.com/posit-dev/positron/blob/e6714474e82205080b98432f605dd8c5da83f67b/extensions/positron-python/src/client/positron/discoverer.ts#L134-L140

Or maybe we could use this directly here?

https://github.com/posit-dev/positron/blob/e6714474e82205080b98432f605dd8c5da83f67b/extensions/positron-python/src/client/interpreter/configuration/environmentTypeComparer.ts#L238-L268